deploy/200-role.yaml

```diff
@@ -82,3 +82,7 @@ rules:
 - apiGroups: ['']
   resources: ['serviceaccounts']
   verbs:     ['get', 'list', 'watch', 'create', 'update', 'delete']
+
+- apiGroups: ['apiextensions.k8s.io']
+  resources: ['customresourcedefinitions', 'customresourcedefinitions/status']
+  verbs:     ['get', 'list', 'watch', 'create', 'update', 'delete', 'patch']
```

Comment:
Can you reduce this. Our controller and the migration script will never create or delete them. I guess overall we might be fine with just `get` and `patch`. Can you try?

Reply:
I tried and it worked.